bio/rbt/fastq_split/wrapper.py

```
@@ -0,0 +1,71 @@
+rbt_output_paths = []
+files_to_compress = {}
+required_threads = 1
+
+# rbt split-fastq does not read compressed fastq files
+read_cmd = f"< {snakemake.input[0]}"
+if str(snakemake.input[0]).endswith("gz"):
+    read_cmd = f"< <(crabz --decompress {snakemake.input[0]})"
+    required_threads += 1
+
+
+for outfile in snakemake.output:
```

Ruff (0.15.21)

[error] Undefined name `snakemake`

(F821)